### PR TITLE
update Chinese

### PR DESCRIPTION
**Pull request by wenzhenxiang**

### CODE REVIEW

File: zh-CN/index.html [hunk 1]
```diff
@@ -120,7 +120,7 @@
                             </li>
                         </ul>
                     </div>
-                    <a href="http://claim.kusama.network/" target="_blank" class="btn btn-outline-primary ml-lg-3">Claim KSM</a>
+                    <a href="http://claim.kusama.network/" target="_blank" class="btn btn-outline-primary ml-lg-3">索取 KSM</a>
```
Comment (xlc) [inline zh-CN/index.html]:
```suggestion
                    <a href="http://claim.kusama.network/" target="_blank" class="btn btn-outline-primary ml-lg-3">领取 KSM</a>
```

File: zh-CN/index.html [hunk 2]
```diff
@@ -154,10 +154,10 @@
                 <div class="col-xl-7 col-lg-8 col-md-9">
                     <div class="d-flex flex-column flex-sm-row justify-content-center mt-4">
                         <a href="http://claim.kusama.network/" class="btn btn-lg btn-cta btn-outline-white text-white mx-2 mb-2 mb-sm-0">
-                            <span class="text-glitch" data-text="CLAIM KSM">CLAIM KSM</span>
+                            <span class="text-glitch" data-text="CLAIM KSM">索取 KSM</span>
```
Comment (xlc) [inline zh-CN/index.html]:
```suggestion
                            <span class="text-glitch" data-text="CLAIM KSM">领取 KSM</span>
```

File: zh-CN/index.html [hunk 3]
```diff
@@ -175,111 +175,111 @@
...
-                                    <p>Kusama is explicitly not a “test net.” It is an early, highly experimental version of Polkadot presenting real economic conditions. The community will own the network – there will be no central kill switch. Kusama will exist as long as its community maintains it and we envision it will cater to new, early functionality and projects preparing to develop and deploy on Polkadot.</p>
+                                    <p>Kusama 是波卡的金丝雀网络，在链下为我们作出危险警报，用户保护开发者的安全。如果没有 Kusama 这样的网络，就没有合理手段来充分了解可能的潜在危险。我们正在建设前沿的实验性技术，这意味着我们无法承诺 Kusama 将做什么或如何工作 — 始于混沌(加一点乐趣)。</p>
+                                    <p>Kusama 显然不是一个“测试网络”。它是波卡一个早期的、高度实验性的版本，呈现了真实的经济状况。社区将拥有该网络-没有中心化禁止开关。只要社区维护它，Kusama 就会存在，我们设想 Kusama 将会服务于新的，早期的功能和准备在波卡上开发和部署的项目。</p>
```
Comment (xlc) [inline zh-CN/index.html]:
```suggestion
                                    <p>Kusama 显然不是一个“测试网络”。它是波卡一个早期的、高度实验性的版本，呈现了真实的经济状况。社区将完全拥有该网络 - 这意味着没有中心化禁止开关。只要社区维护它，Kusama 就会存在，我们设想 Kusama 将会服务于新的，早期的功能和准备在波卡上开发和部署的项目。</p>
```

File: zh-CN/index.html [hunk 4]
```diff
@@ -175,111 +175,111 @@
...
                                     <path d="M13 19L13 5C13 4.44772 12.5523 4 12 4C11.4477 4 11 4.44772 11 5L11 19C11 19.5523 11.4477 20 12 20C12.5523 20 13 19.5523 13 19Z" fill="#212529"></path>
                                 </svg>
                             </div>
                             <div class="collapse" id="panel-3">
                                 <div class="pt-3">
-                                    <p>One percent of DOT tokens at Polkadot genesis time (100,000 of the W3F’s DOTs) have been reserved as an eventual incentivisation to Kusama’s stakeholders and community. The precise mechanics of this have not yet been finalised, but will be firmed up closer to genesis time.</p>
-                                    <p>There is also a standard bounty program for critical issues. You can find out more <a href="https://guide.kusama.network/en/latest/break/bug-bounty/">here</a>.</p>
+                                    <p>在波卡创世期（W3F 的 DOT 中有 100000 个）DOT 代币的百分之一被保留下来，作为对 Kusama 利益相关者和社区的最终激励。其具体机制还未确定，但会在接近创世期时发布。</p>
```
Comment (xlc) [inline zh-CN/index.html]:
```suggestion
                                    <p>在波卡创世期 DOT 代币的百分之一被保留下来（W3F 拥有的 DOT 中的 100000 个），作为对 Kusama 利益相关者和社区的最终激励。其具体机制还未确定，但会在接近创世期时发布。</p>
```

File: zh-CN/index.html [hunk 5]
```diff
@@ -175,111 +175,111 @@
...
                                     <path d="M13 19L13 5C13 4.44772 12.5523 4 12 4C11.4477 4 11 4.44772 11 5L11 19C11 19.5523 11.4477 20 12 20C12.5523 20 13 19.5523 13 19Z" fill="#212529"></path>
                                 </svg>
                             </div>
                             <div class="collapse" id="panel-5">
                                 <div class="pt-3">
-                                    <p>The genesis distribution of KSM will be exactly as for DOT: if you purchased DOTs in the sale then you own an equal share of the Kusama network. The Web3 Foundation will use some of its holdings to fund a frictional faucet for those that do not currently hold DOTs. <a href="https://guide.kusama.network/en/latest/start/claims/">Here’s how to claim KSM</a>.</p>
-                                    <p>Alternatively, a public faucet and a KSM grants process will also be made available upon genesis.</p>
+                                    <p>KSM 创世期的分配与 DOT 完全相同：如果你购买了 DOT，那么你就拥有 Kusama 网络的相同份额。对于那些目前没有持有 DOT 的人，Web3 基金会将用其持有的部分 DOT 来成立一个 KSM 发放池。<a href="https://guide.kusama.network/en/latest/start/claims/">这是如何申请 KSM 的链接</a>。</p>
```
Comment (xlc) [inline zh-CN/index.html]:
```suggestion
                                    <p>KSM 创世期的分配与 DOT 完全相同：如果你购买了 DOT，那么你就拥有 Kusama 网络的相同份额。对于那些目前没有持有 DOT 的人，Web3 基金会将用其持有的部分 DOT 来成立一个 KSM 发放池。<a href="https://guide.kusama.network/en/latest/start/claims/">这是如何领取 KSM 的链接</a>。</p>
```

File: zh-CN/index.html [hunk 6]
```diff
@@ -387,54 +387,54 @@ <h2 class="display-4 mx-xl-6">Kusama?</h2>
...
+                        Kusama 是一个实验性社区的研发网络。
                     </p>
                 </div>
                 <div class="col-6 col-sm-3 col-lg-2 mt-3 mt-lg-0 pl-2">
-                    <h5>Info</h5>
+                    <h5>信息</h5>
                     <ul class="nav flex-column">
                         <li class="nav-item">
-                            <a href="http://claim.kusama.network/" target="_blank" class="nav-link">Claim KSM</a>
+                            <a href="http://claim.kusama.network/" target="_blank" class="nav-link">索取 KSM</a>
```
Comment (xlc) [inline zh-CN/index.html]:
```suggestion
                            <a href="http://claim.kusama.network/" target="_blank" class="nav-link">领取 KSM</a>
```

File: zh-CN/index.html [hunk 7]
```diff
@@ -175,111 +175,111 @@
...
                                     <path d="M13 19L13 5C13 4.44772 12.5523 4 12 4C11.4477 4 11 4.44772 11 5L11 19C11 19.5523 11.4477 20 12 20C12.5523 20 13 19.5523 13 19Z" fill="#212529"></path>
                                 </svg>
                             </div>
                             <div class="collapse" id="panel-3">
                                 <div class="pt-3">
-                                    <p>One percent of DOT tokens at Polkadot genesis time (100,000 of the W3F’s DOTs) have been reserved as an eventual incentivisation to Kusama’s stakeholders and community. The precise mechanics of this have not yet been finalised, but will be firmed up closer to genesis time.</p>
-                                    <p>There is also a standard bounty program for critical issues. You can find out more <a href="https://guide.kusama.network/en/latest/break/bug-bounty/">here</a>.</p>
+                                    <p>在波卡创世期（W3F 的 DOT 中有 100000 个）DOT 代币的百分之一被保留下来，作为对 Kusama 利益相关者和社区的最终激励。其具体机制还未确定，但会在接近创世期时发布。</p>
+                                    <p>对于关键问题也有一个标准的奖励计划。你可以在<a href="https://guide.kusama.network/en/latest/break/bug-bounty/">这里</a>找到更多。</p>
```
Comment (xlc) [inline zh-CN/index.html]:
```suggestion
                                    <p>对于关键问题也有一个标准的赏金计划。你可以在<a href="https://guide.kusama.network/en/latest/break/bug-bounty/">这里</a>找到更多。</p>
```

File: zh-CN/index.html [hunk 8]
```diff
@@ -387,54 +387,54 @@ <h2 class="display-4 mx-xl-6">Kusama?</h2>
...
-                            <a href="https://guide.kusama.network/en/latest/" class="nav-link">User guide</a>
+                            <a href="https://guide.kusama.network/en/latest/" class="nav-link">用户指南</a>
                         </li>
                         <li class="nav-item">
-                            <a href="teams.html" class="nav-link">Teams</a>
+                            <a href="teams.html" class="nav-link">团队成员</a>
                         </li>
                         <li class="nav-item">
-                            <a href="https://guide.kusama.network/en/latest/break/bug-bounty/" class="nav-link">Bug bounty</a>
+                            <a href="https://guide.kusama.network/en/latest/break/bug-bounty/" class="nav-link">Bug 奖励</a>
```
Comment (xlc) [inline zh-CN/index.html]:
```suggestion
                            <a href="https://guide.kusama.network/en/latest/break/bug-bounty/" class="nav-link">Bug 赏金计划</a>
```

**Comment by ansonla3** [comment]
好! 謝謝

**Comment by lsaether** [comment]
@wenzhenxiang @ansonla3 Please resolve conflicts to master before the PR can be merged.

**Pull request merged**